moon/controller_round3.py

```diff
@@ -42,6 +42,16 @@
 
 ATTEMPT_DELAY = timedelta(seconds=30)
 
+def pol2cart(rho, phi):
+    x = rho * math.cos(phi)
+    y = rho * math.sin(phi)
+    return(x, y)
+
+def cart2pol(x, y):
+    rho = np.sqrt(x**2 + y**2)
```

Comment:
is there also `np.hypot()`? https://numpy.org/doc/stable/reference/generated/numpy.hypot.html

Reply:
The formula seems correct as is, what are you suggesting?